docs/tap_hold.md

```diff
@@ -425,6 +425,167 @@ uint16_t get_quick_tap_term(uint16_t keycode, keyrecord_t *record) {
 If `QUICK_TAP_TERM` is set higher than `TAPPING_TERM`, it will default to `TAPPING_TERM`.
 :::
 
+## Chordal Hold
+
+Chordal Hold is intended to be used together with either Permissive Hold or Hold
+On Other Key Press. Chordal Hold is enabled by adding to your `config.h`:
+
+```c
+#define CHORDAL_HOLD
+```
+
+Chordal Hold implements, by default, an "opposite hands" rule. Suppose a
+tap-hold key is pressed and then, before the tapping term, another key is
+pressed. With Chordal Hold, the tap-hold key is settled as tapped if the two
+keys are on the same hand. This behavior may be useful to avoid accidental
+modifier activation with mod-taps, particularly in rolled keypresses when using
+home row mods.
+
+Notes:
+
+* In the case that the keys are on opposite hands, Chordal Hold alone does not
+  yet settle the tap-hold key. Chordal Hold may be used in combination with Hold
+  On Other Key Press or Permissive Hold to determine the behavior. With Hold On
+  Other Key Press, an opposite hands chord is settled immediately as held. Or
+  with Permissive Hold, an opposite hands chord is settled as held provided the
+  other key is pressed and released (nested press) before releasing the tap-hold
+  key.
+
+* Chordal Hold has no effect after the tapping term. 
+
+* Chordal Hold has no effect when the other key is also a tap-hold key. This is
+  so that multiple tap-hold keys may be held on the same hand, which is common
+  to do with home row mods.
+
+* Combos are exempt from the opposite hands rule, since "handedness" is
+  ill-defined in this case. Even so, Chordal Hold's behavior involving combos
+  may be customized through the `get_chordal_hold()` callback.
+
+An example of a sequence that is affected by “chordal hold”: 
+
+- `SFT_T(KC_A)` Down
+- `KC_C` Down
+- `KC_C` Up
+- `SFT_T(KC_A)` Up
+
+```
+                         TAPPING_TERM   
+  +---------------------------|--------+
+  | +----------------------+  |        |
+  | | SFT_T(KC_A)          |  |        |
+  | +----------------------+  |        |
+  |    +--------------+       |        |
+  |    | KC_C         |       |        |
+  |    +--------------+       |        |
+  +---------------------------|--------+
+```
+
+If the two keys are on the same hand, then this will produce `ac` with 
+`SFT_T(KC_A)` settled as tapped the moment that `KC_C` is pressed. 
+
+If the two keys are on opposite hands and the `HOLD_ON_OTHER_KEY_PRESS`
+option enabled, this will produce `C` with `SFT_T(KC_A)` settled as held the
+moment that `KC_C` is pressed.
+
+Or if the two keys are on opposite hands and the `PERMISSIVE_HOLD` option is
+enabled, this will produce `C` with `SFT_T(KC_A)` settled as held the
+moment that `KC_C` is released.
```

Comment:
Opposite hand behavior is expected to follow the configurations specified in the previous sections. To avoid confusion, we should clarify that Chordal Hold _does not_ introduce new behavior for opposite keys with _hold on other key press_ or _permissive hold_.

Reply:
Ah, thanks for calling that out! I've reworded to say this explicitly. Yes, the intention is that Chordal Hold settles same hand chords as tapped, while behavior is unchanged for opposite hands chords. New wording:

```
Chordal Hold implements, by default, an "opposite hands" rule. Suppose a
tap-hold key is pressed and then, before the tapping term, another key is
pressed. With Chordal Hold, the tap-hold key is settled as tapped if the two
keys are on the same hand.

Otherwise, if the keys are on opposite hands, Chordal Hold introduces no new
behavior. Hold On Other Key Press or Permissive Hold may be used together with
Chordal Hold to configure the behavior in the opposite hands case. With Hold On
Other Key Press, an opposite hands chord is settled immediately as held. Or with
Permissive Hold, an opposite hands chord is settled as held provided the other
key is pressed and released (nested press) before releasing the tap-hold key.
```

I originally considered that Chordal Hold could settle the opposite hands chords as held. I think leaving them unchanged is a better design though, since this way there is more flexibility to use other tap-hold options to customize the behavior.